tests/model_serving/model_server/utils.py

```diff
@@ -84,6 +84,12 @@ def verify_inference_response(
             reason = "Forbidden"
             assert reason in res["output"], f"{reason} not found in output:\n{res['output']}"
 
+        elif (
+            isinstance(inference_service, InferenceGraph)
+            and inference.deployment_mode == KServeDeploymentType.RAW_DEPLOYMENT
+        ):
```

Comment:
what about rawDeployment for inferenceGraph?

Reply:
This if branch covers that, no?

Reply:
Found a logic for serverless.